src/map/itemdb.c

```diff
@@ -1299,6 +1309,71 @@ void itemdb_read_packages(void) {
 	ShowStatus("Done reading '"CL_WHITE"%d"CL_RESET"' entries in '"CL_WHITE"%s"CL_RESET"'.\n", count, config_filename);
 }
 
+void itemdb_read_options(void)
+{
+	struct config_t item_options_db;
+	struct config_setting_t *ito = NULL, *conf = NULL;
+	int index = 0, count = 0;
+	const char *filepath = "db/item_options.conf";
+	
+	if (!libconfig->load_file(&item_options_db, filepath))
+		return;
+	
+#ifdef ENABLE_CASE_CHECK
+	script->parser_current_file = filepath;
+#endif // ENABLE_CASE_CHECK
+	
+	if ((ito=libconfig->setting_get_member(item_options_db.root, "item_options_db")) == NULL) {
+		ShowError("itemdb_read_options: '%s' could not be loaded.\n", filepath);
+		libconfig->destroy(&item_options_db);
+		return;
+	}
+	
+	while ((conf = libconfig->setting_get_elem(ito, index++))) {
+		struct item_option t_opt = { 0 }, *s_opt = NULL;
+		const char *str = NULL;
+		
+		if (!libconfig->setting_lookup_int16(conf, "Id", &t_opt.index) || t_opt.index <= 0) {
+			ShowError("itemdb_read_options: Invalid Option Id provided for entry %d in '%s', skipping...\n", t_opt.index, filepath);
+			continue;
+		}
+		
+		if (!libconfig->setting_lookup_string(conf, "Name", &str)) {
+			ShowError("itemdb_read_options: Invalid Option variable name provided for Option Id %d in '%s', skipping...\n", t_opt.index, filepath);
+			continue;
+		}
+		
+		/* Set Constant */
+		script->set_constant(str, t_opt.index, false, false);
+		
+		if (!libconfig->setting_lookup_string(conf, "Script", &str)) {
+			ShowError("itemdb_read_options: Script code not found for entry %s (Id: %d) in '%s', skipping...\n", str, t_opt.index, filepath);
+			continue;
+		}
+		
+		/* Set Script */
+		t_opt.script = *str ? script->parse(str, filepath, t_opt.index, SCRIPT_IGNORE_EXTERNAL_BRACKETS, NULL) : NULL;
+		
```

Comment:
Please add here call to function for read additional fields. For example like in mob.c mob->read_db_additional_fields.
This can be usefull for plugins too